reco_utils/recommender/deeprec/config/lightgcn.yaml

```diff
@@ -0,0 +1,22 @@
+#model
+model:
+    model_type : "lightgcn"
+    embed_size : 64
+    n_layers : 3
+    
+#train
+train:
+    batch_size : 1024
+    decay : 0.0001
+    epochs : 1000
+    learning_rate : 0.001
+    eval_epoch : -1
```

Comment:
Please give some commentary to the parameters, so readers can follow and change setting more easily.

Reply:
Sorry, I forgot that😅